e2e/tests/overview-page.spec.ts

```diff
@@ -241,6 +241,28 @@ test('Export to CSV button fails to request all features and shows toast', async
   await toast.waitFor({state: 'visible'});
 });
 
+test('Test id search atoms in a query', async ({page}) => {
+  await gotoOverviewPageUrl(page, 'http://localhost:5555/');
+  const searchbox = page.locator('#inputfield');
+  await expect(searchbox).toBeVisible();
+  await expect(searchbox).toHaveAttribute('value', '');
+
+  const stats = await page.locator('.stats-summary');
+  const textContent = await stats.innerText();
+  expect(parseInt(textContent.split(' ')[0])).toBeGreaterThan(7);
+
+  const sevenIDAtoms =
+    'id:Molestiae77 OR id:Ratione74 OR id:Molestias63 OR id:Ut59 OR id:Ad50 OR id:Inventore43 OR id:Rem51';
+  await page.keyboard.type('/' + sevenIDAtoms);
+  await expect(searchbox).toHaveAttribute('value', sevenIDAtoms);
+  await page.locator('#filter-submit-button').click();
+  await page.waitForTimeout(1000);
```

Comment:
We have a way to wait for the loading to finish on the overview page:

https://github.com/GoogleChrome/webstatus.dev/blob/ecb10ffb58d0df1557c6b04ca68cb8b42d7ee692/e2e/tests/utils.ts#L58-L61